lib/misc.c

```diff
@@ -162,6 +162,13 @@ void strip_html(char *in)
 
 	memset(out, 0, sizeof(out));
 
+	if (strlen(in) == 0) {
```

Comment:
It's reading the first byte of the string VS calling one function which may iterate all the way to the end of the string to return a length that isn't used. Unlikely to be a big deal, so i'd rather do it for the sake of doing it the same way as the block below.

(That being said, see my other comment, the whole thing might be wrong :D)